Card/src/main/java/com/braintreepayments/api/BaseCard.java

```diff
@@ -32,20 +32,21 @@
     static final String REGION_KEY = "region";
     static final String STREET_ADDRESS_KEY = "streetAddress";
 
-    protected String cardholderName;
```

Comment:
When these properties are `protected` and set in Kotlin using the properties like `card.cardHolderName = "some name"`, the underlying setter method is not invoked for some reason, and the properties are set directly. This means that the conversion from empty string to `null` doesn't happen. This led to test failures in the tests around not adding emptying strings to the `json` objects. Updating these to be `private` and using the getters to access them in `Card` and `UnionPayCard` subclasses resolved the problem.